src/views/System/Settings/Index.vue

```diff
@@ -39,15 +43,15 @@ export default {
   components: {
     CSystemEditorAuth,
     CSystemEditorExternal,
+    CSystemEditorAuthBgScreen,
   },
 
-  mixins: [
-    editorHelpers,
-  ],
+  mixins: [editorHelpers],
 
   data () {
     return {
       settings: [],
+      bgSettings: {},
```

Comment:
no need for extra property since when you fetch you recompute it anyways (similar to getAuth)